WordPress/WordPressTest/PostTests.swift

```diff
@@ -310,9 +310,18 @@ class PostTests: XCTestCase {
 
     func testThatHasLocalChangesWorks() {
         let original = newTestPost()
-        let revision = original.createRevision() as! Post
+        var revision = original.createRevision() as! Post
 
-        XCTAssertFalse(original.hasLocalChanges())
```

Comment:
Since there's nothing stopping developers from calling `hasLocalChanges()` on "original" posts, the test removed in this line still covers a valid execution path.

As long as we allow that execution path, I'd keep this test.  It can be separated if we wanted to keep things atomic, though - but I'd avoid deleting it.